### PR TITLE
feat(elements|ino-autocomplete): provide keyboard support to open menu

### LINKED ISSUE #443: ino-autocomplete: Open menu with keyboard
After selecting an option with keyboard (pressing "Enter" key), it is necessary to refocus the input to open the menu. To improve this behavior I suggest opening the menu by using the arrow keys or typing something  in input.

### PR DESCRIPTION
**Pull request by BenPag**
Closes #443 

## Proposed Changes

- open menu by keyboard press after option selection

### CODE REVIEW

File: packages/elements/src/components/ino-autocomplete/ino-autocomplete.tsx [hunk 1]
```diff
@@ -115,13 +115,18 @@ export class Autocomplete implements ComponentInterface {
...
-      !this.menuIsVisible ||
       !this.filteredOptionEls ||
-      this.filteredOptionEls.length === 0
+      this.filteredOptionEls.length === 0 ||
+      (this.lastKeyCode !== 'Enter' && !this.menuIsVisible)
     ) {
       return;
     }
 
+    if (this.lastKeyCode === 'Enter' && !this.menuIsVisible) {
```
[the remark this answers is not in the document]
Comment (BenPag) [reply packages/elements/src/components/ino-autocomplete/ino-autocomplete.tsx]: Alright, thats good. I switched the order of the statements so you can open for instance with `ArrowDown` the menu. But I don´t know if this is a good practise.

[the remark this answers is not in the document]
Comment (janivo) [reply packages/elements/src/components/ino-autocomplete/ino-autocomplete.tsx]: The `default` case in the middle looks kinda difficult to grasp. I would rather put it at the bottom of the switch and call the `openMenu()` method in the arrow cases

**Review by janivo — COMMENTED** [review]
Works nicely 👍🏼 
Only two things I noticed:
- Does it make sense to call `openMenu()` on `Enter` press?
- Whenever I select a value, delete the input, and blur the input, the value `null` is selected

**Comment by BenPag** [comment]
> Works nicely 👍🏼 Only two things I noticed:
> 
> * Does it make sense to call `openMenu()` on `Enter` press?
> * Whenever I select a value, delete the input, and blur the input, the value `null` is selected

To the first point I removed the `openMenu()` call.
The second thing is in my point of view a right behavior, do you expected something else?

**Comment by janivo** [comment]
> > Works nicely 👍🏼 Only two things I noticed:
> > 
> > * Does it make sense to call `openMenu()` on `Enter` press?
> > * Whenever I select a value, delete the input, and blur the input, the value `null` is selected
> 
> To the first point I removed the `openMenu()` call. The second thing is in my point of view a right behavior, do you expected something else?

Now that I think about it, you are right, that should be the behavior. 
But we should document that the event could return `null` by using the correct typing for the event emitter like this: `EventEmitter<string | null>`.
Also, we should check that `null` is returned and not `"null"` as a string. Maybe we should add a test case for this one?

**Comment by BenPag** [comment]
I add a new test case and change the emetter type to `EventEmitter<string | null>.`

**Review by janivo — APPROVED** [review]
